tests/containers/bci_test.pm

```diff
@@ -72,17 +72,16 @@ sub run {
     assert_script_run("export BCI_DEVEL_REPO=$bci_devel_repo") if $bci_devel_repo;
 
     # Run the tests for each environment
-    my $cmd_options = check_var('HOST_VERSION', '12-SP5') ? '' : '-- -n auto';
```

Comment:
So you've dropped `-n auto` unconditionally?

Reply:
oops, I actually wanted the opposite... (facepalm)... wait for it.